article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -61,68 +66,82 @@ case class ReaderRevenueLinks(
   ampFooter: ReaderRevenueLink
 )
 
-case class PageData(
-    author: String,
-    pageId: String,
-    pillar: Option[String],
-    ajaxUrl: String,
-    webPublicationDate: Long,
-    webPublicationDateDisplay: String,
-    section: Option[String],
-    sectionLabel: String,
-    sectionUrl: String,
-    headline: String,
-    webTitle: String,
-    byline: String,
-    contentId: Option[String],
-    authorIds: Option[String],
-    keywordIds: Option[String],
-    toneIds: Option[String],
-    seriesId: Option[String],
-    isHosted: Boolean,
-    beaconUrl: String,
-    editionId: String,
-    edition: String,
-    contentType: Option[String],
-    commissioningDesks: Option[String],
-    subMetaLinks: SubMetaLinks,
-    sentryHost: String,
-    sentryPublicApiKey: String,
-    switches: Map[String,Boolean],
-    linkedData: List[LinkedData],
-    subscribeWithGoogleApiUrl: String,
-
-    // AMP specific
-    guardianBaseURL: String,
-    webURL: String,
-    shouldHideAds: Boolean,
-    hasStoryPackage: Boolean,
-    hasRelated: Boolean,
-    isCommentable: Boolean,
-    commercialProperties: Option[CommercialProperties],
-    starRating: Option[Int],
-    trailText: String,
+case class Meta(
+  isImmersive: Boolean,
+  isHosted: Boolean,
+  shouldHideAds: Boolean,
+  hasStoryPackage: Boolean,
+  hasRelated: Boolean,
+  isCommentable: Boolean,
+  linkedData: List[LinkedData]
+)
+
+case class Tags(
+  authorIds: Option[String],
+  toneIds: Option[String],
+  keywordIds: Option[String],
+  commissioningDesks: Option[String],
+  all: List[Tag]
 )
 
-case class Config(
-    isImmersive: Boolean,
-    page: PageData,
-    nav: NavMenu,
-    readerRevenueLinks: ReaderRevenueLinks
+// the big three
+
+case class DCConfig(
+  ajaxUrl: String,
+  guardianBaseURL: String,
+  sentryHost: Option[String],
+  sentryPublicApiKey: Option[String],
+  switches: Map[String,Boolean],
+  beaconUrl: String,
+  subscribeWithGoogleApiUrl: String
+)
+
+case class DCSite(
+  nav: NavMenu,
+  readerRevenueLinks: ReaderRevenueLinks
+)
+
+case class Content(
+  headline: String,
+  standfirst: Option[String],
+  main: String,
+  body: String,
+  blocks: Blocks,
+  byline: String,
+  trailText: String
 )
```

Comment:
Shall we move this up, to keep _The Big Three_ together?

Reply:
Big two now, but yes I've moved this so page/site classes are together